### PR TITLE
doc/man/8/rados.rst: including all available options

### PR DESCRIPTION
**Pull request by rs-fabrica**
Including all available options in rados command line utility man page.

### CODE REVIEW

File: doc/man/8/rados.rst [hunk 1]
```diff
@@ -19,13 +19,21 @@ Description
 cluster (RADOS), part of the Ceph distributed storage system.
 
 
-Options
-=======
+Global Options
+==============
+
+.. option:: --object_locator object_locator
```
Comment (tchaikov) [inline doc/man/8/rados.rst]:
```suggestion
.. option:: --object-locator object_locator
```
more consistent this way.

File: doc/man/8/rados.rst [hunk 2]
```diff
@@ -69,18 +85,154 @@ Options
 
    Uses the striping API of rados rather than the default one.
    Available for stat, stat2, get, put, append, truncate, rm, ls
-   and all xattr related operation
+   and all xattr related operation.
+
+.. option:: -o object_size
```
Comment (tchaikov) [inline doc/man/8/rados.rst]:
```suggestion
.. option:: -O object_size, --object-size object_size
```

File: doc/man/8/rados.rst [hunk 3]
```diff
@@ -69,18 +85,154 @@ Options
...
 
    Will set the lock cookie for acquiring advisory lock (lock get command).
    If the cookie is not empty, this option must be passed to lock break command
    to find the correct lock when releasing lock.
 
+.. option:: --target-locator
+
+   Use with cp to specify the locator of the new object.
+
+   .. option:: --target-nspace
```
Comment (tchaikov) [inline doc/man/8/rados.rst]: wrong indent?

File: doc/man/8/rados.rst [hunk 4]
```diff
@@ -69,18 +85,154 @@ Options
...
+
+   Use with cp to specify the namespace of the new object.
+
+
+Bench options
+=============
+
+   .. option:: -t N, --concurrent-ios=N
```
Comment (tchaikov) [inline doc/man/8/rados.rst]: why do you start to indent the options here?

File: doc/man/8/rados.rst [hunk 5]
```diff
@@ -69,18 +85,154 @@ Options
...
+
+   .. option:: --omap-key-file file
+
+   Read the omap key from a file.
+
+
+Generic options
+===============
+
+   .. option:: -c FILE, --conf FILE
```
Comment (tchaikov) [inline doc/man/8/rados.rst]:
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/man/8/rados.rst:209: WARNING: Duplicate explicit target name: "cmdoption-rados-c".
```

File: doc/man/8/rados.rst (hunk 3 above)
Comment (rs-fabrica) [reply doc/man/8/rados.rst]: Yes.

File: doc/man/8/rados.rst (hunk 4 above)
Comment (rs-fabrica) [reply doc/man/8/rados.rst]: My mistake. Commiting the fix.

File: doc/man/8/rados.rst (hunk 5 above)
Comment (rs-fabrica) [reply doc/man/8/rados.rst]: Found the duplication.

Fix commited.

**Comment by tchaikov** [comment]
@rs-fabrica could you please address all comments?

File: doc/man/8/rados.rst (hunk 3 above)
Comment (rs-fabrica) [reply doc/man/8/rados.rst]: Fix commited

Comment (rs-fabrica) [reply doc/man/8/rados.rst]: Fix commited

**Comment by rs-fabrica** [comment]
I'm sorry, I didn't realize there was a "Resolve conversation" button.

I believe everything is fine now.

**Comment by tchaikov** [comment]
@rs-fabrica could you squash these commits into a single one?

**Comment by rs-fabrica** [comment]
Squashed.

File: doc/man/8/rados.rst [hunk 6]
```diff
@@ -69,18 +80,154 @@ Options
...
+
+OMAP options
+============
+
+.. option:: --omap-key-file file
+
+   Read the omap key from a file.
+
+
+Generic options
```
Comment (tchaikov) [inline doc/man/8/rados.rst]: what's the difference between "global options" and "generic options"?

Comment (tchaikov) [reply doc/man/8/rados.rst]: i see .. it's the options mentioned by `generic_usage()`.

**Pull request merged**